compose/ui/ui/src/skikoTest/kotlin/androidx/compose/ui/window/PopupTest.kt

```diff
@@ -599,6 +599,40 @@ class PopupTest {
         )
     }
 
+    @Test
+    fun secondaryButtonClickDismissPopup() = runSkikoComposeUiTest(
+        size = Size(100f, 100f)
+    ) {
+        val openPopup = mutableStateOf(true)
+        val background = FillBox()
+        val popup = PopupState(
+            IntRect(20, 20, 60, 60),
+            focusable = true,
+            onDismissRequest = {
+                openPopup.value = false
+            }
+        )
+
+        setContent {
+            background.Content()
+            if (openPopup.value) {
+                popup.Content()
+            }
+        }
+
+        val buttons = PointerButtons(
+            isSecondaryPressed = true
+        )
+        scene.sendPointerEvent(
+            PointerEventType.Press,
+            position = Offset(10f, 10f),
+            buttons = buttons,
+            button = PointerButton.Secondary
+        )
+
```

Comment:
Should send a release here too

Reply:
No, it's on purpose - `Popup` should be closed on down event

Reply:
So dialogs close on click, but popups close on press? I see that in the code, but is there a reason for it?

Reply:
Yes. TLDR - Android